components/cv_form/personal_info.tsx

```diff
@@ -76,17 +52,22 @@ export function PersonalInfo({ fProps }: Props) {
           </span>
         </div>
         <div className="md:flex-grow">
-          <select
+          <Field
+            as="select"
+            name="title_id"
             className="rounded-md"
-            value={title}
-            onChange={handleTitleChange}
+            defaultValue={fProps.values.title_id || ""}
```

Comment:
This should ensure that initially, on edit page proper title will be preselected and on new page it should preselect disabled option "Choose title".